.expeditor/scripts/verify/build_package-aarch64.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eou pipefail
+
+package_path=${1?package_path argument required}
+
+# Install hab from a temporarily uploaded aarch64 package
+curl https://raw.githubusercontent.com/habitat-sh/habitat/main/components/hab/install.sh | sudo bash -s -- -t "$BUILD_PKG_TARGET" -c "$HAB_FALLBACK_CHANNEL" -v 1.6.1178
```

Comment:
could we just put this package in stable and not need to rely on a specific build number? I don't see a problem with that since we have not publicized arm support and it wouldn't be possible for customers to see it on the official downloads site.

Reply:
Once this is merged, I am thinking of updating the channel to dev to build the latest hab and update the pipeline. If that doesn’t work, I can promote the build created by the release pipeline and then update it here.